raiden/transfermanager.py

```diff
@@ -86,68 +128,172 @@ def transfer_async(self, amount, target, identifier=None, callback=None):
         """
 
         # Create a default identifier value
-        if not identifier:
+        if identifier is None:
```

Comment:
yes I agree as discussed in the chat. Let's leave it as is (with Augusuto's change applied that is).